src/tests/fs_tests.rs

```diff
@@ -2104,4 +2111,65 @@ pub mod fs_tests {
         assert_eq!(cage.exit_syscall(EXIT_SUCCESS), EXIT_SUCCESS);
         lindrustfinalize();
     }
+
+    #[test]
+    pub fn ut_lind_fs_creat_new_file() {
```

Comment:
Since this syscall is similar to `open_syscall`, added tests for covering the happy cases only.